src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 7) {
+    total = total - 50;
+  }
+
+  if (days >= 3 && days < 7) {
+    total = total - 20;
+  }
```

Comment:
40, 50, 20 - magic numbers for now
Will be better to create variables to store these values and use these variables in code

Reply:
point 1 in [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)